README.md

```diff
@@ -36,3 +36,26 @@ Example:
 ```sh
 $ appmap prune appland.results.json 2MB
 ```
+
+### How it works
+Pruning works by finding the most repetitive calls within a given "chunk" and
+removing events associated with those calls. A chunk is defined as a group of
```

Comment:
I'd change "chunk is defined as" to "chunk is identified as"

Reply:
I use `defined as` because this is new term that I'm introducing for the purpose of explanation. To me, `identified as` portrays the use of an established or well-known term.